apps/remix-ide/src/lib/offsetToLineColumnConverter.js

```diff
@@ -36,6 +36,19 @@ export class OffsetToLineColumnConverter extends Plugin {
     return this.sourceMappingDecoder.convertOffsetToLineColumn(rawLocation, this.lineBreakPositionsByContent[file])
   }
 
+  offsetToLineColumnWithContent (rawLocation, file, sources, content) {
```

Comment:
it looks like this one is too complex.
`sources` might not really be needed. can be resumed too
```
offsetToLineColumnWithContent (rawLocation, file, content) {
    if (!this.lineBreakPositionsByContent[file]) {
      this.lineBreakPositionsByContent[file] = this.sourceMappingDecoder.getLinebreakPositions(content)
    }
    return this.sourceMappingDecoder.convertOffsetToLineColumn(rawLocation, this.lineBreakPositionsByContent[file])
  }
```